core/src/main/kotlin/com/alecstrong/sql/psi/core/sql.bnf

```diff
@@ -336,14 +336,17 @@ compound_select_stmt ::= [ with_clause ] select_stmt  ( compound_operator select
   ]
 }
 limiting_term ::= expr
-select_stmt ::= SELECT [ DISTINCT | ALL ] result_column ( COMMA result_column ) * [ FROM join_clause ] [ WHERE expr ] [ GROUP BY expr ( COMMA expr ) * [ HAVING expr ] ] | VALUES values_expression ( COMMA values_expression ) * {
+select_stmt ::= SELECT [ DISTINCT | ALL ] result_column ( COMMA result_column ) * [ FROM join_clause ] [ WHERE expr ] [groupBy] [HAVING expr] | VALUES values_expression ( COMMA values_expression ) * {
   mixin = "com.alecstrong.sql.psi.core.psi.mixins.SelectStmtMixin"
   implements = [
     "com.alecstrong.sql.psi.core.psi.QueryElement";
     "com.alecstrong.sql.psi.core.psi.SqlCompositeElement"
   ]
   pin = 1
 }
+
+groupBy ::= GROUP BY expr ( COMMA expr ) *
```

Comment:
Split to allow overwriting it in postgresql GROUP BY ROLLUP (comes in another PR)